modules/stanford_earth_migrate_extend/src/Plugin/migrate/process/StanfordEarthFeedDept.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\stanford_earth_migrate_extend\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Looks up the Department or Program that goes with the events feed URL.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "stanford_earth_feed_dept"
+ * )
+ */
+class StanfordEarthFeedDept extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    $department = NULL;
+    $properties = [
+      'name' => $row->getSourceProperty('current_feed_url'),
+      'vid' => 'stanford_earth_event_feeds',
+    ];
+    $terms = \Drupal::entityTypeManager()
```

Comment:
This should be injected.

Reply:
This followed the example of the migrate_file module's FileImport process plugin which uses the \Drupal::entityTypeManager() code. Fortunately, I found an example of doing this with DI in the core menu_link_content module's LinkUri process plugin so I'll add that change.